tests/decode-nsh-type1/test.yaml

```diff
@@ -0,0 +1,19 @@
+# *** Add configuration here ***
```

Comment:
You can remove this header line

Reply:
> You can remove this header line

@catenacyber Do I need something this in test.yaml as well?

`requires:
    min-version: 6
`

Reply:
Yes indeed, as NSH is not yet in version 5